test/code/plugins/squid_log_parser_plugintest.rb

```diff
@@ -0,0 +1,64 @@
+require 'fluent/test'
+require 'fluent/test/parser_test'
+require_relative '../../../source/code/plugins/squidlogparser.rb '
+require_relative 'omstestlib'
+
+class squidlogParserTest < Test::Unit::TestCase
```

Comment:
All class names in Ruby must be capitalized: SquidLogParserTest;
See line 9339 in the Travis output here: https://travis-ci.org/Microsoft/OMS-Agent-for-Linux/builds/200542080